src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -92,9 +92,6 @@ public CitizenManager(final Colony colony)
     @Override
     public void read(@NotNull final CompoundNBT compound)
     {
-        maxCitizens = compound.getInt(TAG_MAX_CITIZENS);
-        potentialMaxCitizens = compound.getInt(TAG_POTENTIAL_MAX_CITIZENS);
```

Comment:
Shouldn't we then recalc after read?

Reply:
ye but we need the buildings to be loaded for that calculation, thats why it is added after the building manager read in the colony

Reply:
ahkk